drivers/interrupt_controller/intc_plic.c

```diff
@@ -29,7 +29,7 @@
 #define PLIC_IRQS        (CONFIG_NUM_IRQS - CONFIG_2ND_LVL_ISR_TBL_OFFSET)
 #define PLIC_EN_SIZE     ((PLIC_IRQS >> 5) + 1)
 
-#define PLIC_EDGE_TRIG_TYPE   (PLIC_MAX_PRIO + DT_INST_PROP(0, riscv_trigger_reg_offset))
```

Comment:
Maybe this `PLIC_MAX_PRIOR` should have been the base address instead?

Reply:
http://www.andestech.com/wp-content/uploads/AX45MP-1C-Rev.-5.0.0-Datasheet.pdf

Looks like it's something like
`trig = base + n * sizeof(uint32_t) + n / BITS_PER_BYTE`, where `n = NHPOT(num_irqs)`.

Reply:
Trigger configuration is mentioned in the PLIC spec, but it looks like the offset was omitted from the memory map.

https://github.com/riscv/riscv-plic-spec/blob/master/riscv-plic.adoc#memory-map

Reply:
> Looks like it's something like `trig = base + n * sizeof(uint32_t) + n / BITS_PER_BYTE`, where `n = NHPOT(num_irqs)`.

I think my assumption about the offsets being based on the number of routed interrupts was wrong. E.g. https://bit.ly/3r5PDS4 only incorporates 63 of the possible 1023 and still uses an offset of 0x1080 (which makes sense in a lot of ways).